packages/piro/src/Piro_InvertMassMatrixDecorator.hpp

```diff
@@ -55,10 +55,14 @@
 #include "Kokkos_DefaultNode.hpp"
 #endif
 
+#include "Tpetra_Map.hpp"
+using default_lo = Tpetra::Map<>::local_ordinal_type;
+using default_go = Tpetra::Map<>::global_ordinal_type;
+
 namespace Piro {
 
 #ifdef ALBANY_BUILD
-template <typename Scalar, typename LocalOrdinal = int, typename GlobalOrdinal = LocalOrdinal,
+template <typename Scalar, typename LocalOrdinal = default_lo, typename GlobalOrdinal = default_go,
           typename Node = KokkosClassic::DefaultNode::DefaultNodeType>
```

Comment:
Did you also want to make the default Node type in Piro a type alias, as with MueLu above?